cypress/integration/instance/InstanceEdit.ts

```diff
@@ -50,5 +42,39 @@ onlyOn(isEnvironmentType(EnvType.Mocked), () => {
             });
         });
     });
+
+    it("Switch from 'Webhook' to 'Ignore' Error handler", () => {
+      // Switch to error handling tab
+      cy.ouiaId("error-handling", "PF4/TabButton").click();
```

Comment:
It depends on you but this line is same for both tests. It could be move to `beforeEach` method.

Reply:
pushed in a separate commit 1400d3c